### PR TITLE
DataModifier.types demo

### PR DESCRIPTION
**Pull request by bre1470**
Based on #18642

### CODE REVIEW

File: samples/highcharts/data-tools/datamodifier-tree/demo.js [hunk 1]
```diff
@@ -0,0 +1,94 @@
+const sortModifier = new Highcharts.DataModifier.types.Sort();
```
Comment (karolkolodziej) [inline samples/highcharts/data-tools/datamodifier-tree/demo.js:1]: Error in the console.

**Review by karolkolodziej — CHANGES_REQUESTED** [review]

File: samples/highcharts/data-tools/datamodifier-tree/demo.js (hunk 1 above)
Comment (pawelfus) [reply samples/highcharts/data-tools/datamodifier-tree/demo.js:1]: Works fine on my branch, did you build it?

**Review by pawelfus — CHANGES_REQUESTED** [review]
Just a minor polishing, thanks!

File: samples/highcharts/data-tools/datamodifier-tree/demo.js (hunk 1 above)
Comment (karolkolodziej) [reply samples/highcharts/data-tools/datamodifier-tree/demo.js:1]: Yep, now works ;)

**Review by karolkolodziej — APPROVED** [review]

**Review by pawellysy — APPROVED** [review]
Great work! Thanks!

**Review by TorsteinHonsi — APPROVED** [review]
Thanks!

**Review by sebastianbochan — APPROVED** [review]

**Pull request merged**